docs/index.rst

```diff
@@ -12,6 +12,11 @@ way, which separates extraction and crawling logic from the I/O,
 and from the Scrapy implementation details as well.
 It makes the code more testable and reusable.
 
+Beside, it
```

Comment:
I think this shouldn't be a new paragraph, as it's linked to the idea of the previous one.

Also, but I'm not sure about this one.
_Beside, it opens .._ -> _Furthermore, it opens .._

Reply:
Spot on! Changed here https://github.com/scrapinghub/scrapy-poet/pull/43/commits/948885f1fdd891bd24b329259435308af82d3875